msteams-platform/bots/how-to/conversations/bot-copilot-handoff.md

```diff
@@ -0,0 +1,178 @@
+   |`continuationToken`     | The `continuationToken` value can be any string, but ensure that the total URL length doesn't exceed 2048 characters and the string is URL encoded.        |
+   |`Action.OpenUrl`     |  The `Action.OpenUrl` property allows the user to handoff the conversation to a bot.       |
+
+   **Sample payload**
+
+   ```json
+   { 
+    "name": "handoff/action", 
+    "type": "invoke", 
+    "timestamp": "2024-04-15T19:50:32.945Z", 
+    "localTimestamp": "2024-04-15T19:50:32.945Z", 
+    "id": "f:ad06278a-0dd1-8811-71b1-f65e2bfd4570", 
+    "channelId": "msteams", 
+    "serviceUrl": "https://smba.trafficmanager.net/amer/", 
+    "from": { 
+        "id": "29:1jzORtjcfpYTLQDR9O4TyLz9LDwHskubQN1Ljc-aFO4L8dnZatjFpSw1PCGa-Mm-Jo4uLp67Lvekcjq2hkPoxdA", 
+        "aadObjectId": "57162b23-ae6f-4e95-9774-27f7cb019864" 
+    }, 
+    "conversation": { 
+        "conversationType": "personal", 
+        "tenantId": "72f988bf-86f1-41af-91ab-2d7cd011db47", 
+        "id": "a:13tOiSzRqeub3zaqoTHKpvOkk8Y1zFxk-g8WKdAUM2tjhTBFMt4RSuL8YWi7uwFNBmbxsyzYYktJEyfimYXYiEoplQ34aJs1y8trDb7EIcG09xOjSUieHVzFZ2b8tkagZ" 
+    }, 
+    "recipient": { 
+        "id": "28:68935e91-ff09-4a33-a675-0fe09f015706", 
+        "name": "NorthwindProducts" 
+    }, 
+    "entities": [ 
+        { 
+            "locale": "en-US", 
+            "country": "US", 
+            "platform": "Android", 
+            "timezone": "America/Chicago", 
+            "type": "clientInfo" 
+        } 
+    ], 
+    "channelData": { 
+        "tenant": { 
+            "id": "72f988bf-86f1-41af-91ab-2d7cd011db47" 
+        }, 
+        "source": { 
+            "name": "message" 
+        }, 
+        "legacy": { 
+            "replyToId": "1:1_qLAAGcfze29QAWxzicc7gvR3vuNAlKvth08vavxYYs" 
+        } 
+    }, 
+    "replyToId": "1713210583687", 
+    "value": { 
+        "continuation": "test-continuation-token" 
+    }, 
+    "locale": "en-US", 
+    "localTimezone": "America/Chicago", 
+    "rawTimestamp": "2024-04-15T19:50:32.945Z", 
+    "rawLocalTimestamp": "2024-04-15T14:50:32.945-05:00", 
+    "callerId": "urn:botframework:azure" 
+   }
+   ```
+
+   The `Action.OpenUrl` property allows the user to handoff the conversation to a bot.  After the user is redirected to the bot chat, the bot receives the `continuationToken` value as an invoke message of the type `handoff/action`, if the app associated with the bot is installed in the user's personal scope and the continuationToken value isn't empty.
+
+1. **Handle the invoke type**: In your bot code, manage the `handoff/action` invoke using the `onInvokeActivity` handler. Override the `onInvokeActivity` method to handle the invoke call. The response to this invoke call must be an HTTP status code 200 for success or a code in the range of 400-500 for error.
```

Comment:
What happens for other status codes?

Are we strictly saying 400-500 only? What about 502? 503?

Reply:
What does the user see if an error status code is returned?

Reply:
How about `or a (400- or 500-level) error code response` (in place of "or a code in the range of 400-500 for error")